### ISSUE TITLE
Full workbench theming

### ISSUE THREAD

**Issue by kurtextrem** [equinusocio/vsc-material-theme#12]
https://twitter.com/BenjaminPasero/status/847582977198510081

So I've opened this issue in advance, to ask whether you will be bringing in the ST3 background color of the Material Theme :)

**Comment by equinusocio** [comment]
Thank you, i'm just try to find some documentation about that, no informations in the Insiders build

**Comment by kurtextrem** [comment]
I think it's not live yet - will report back if I see a tweet that it is.
Just put this issue up for the future

**Comment by equinusocio** [comment]
https://twitter.com/code/status/847918623205113857

**Comment by equinusocio** [comment]
More info here:
https://github.com/Microsoft/vscode/blob/ea81c98e6d5c089fd97ebcd871b1353bcf79f0a6/extensions/theme-abyss/themes/abyss-color-theme.json

```json
"colors": {

		// Base Colors
		"inputBoxBackground": "#181f2f",
		"dropdownBackground": "#181f2f",
		"inputBoxActiveOptionBorder": "#266fa0",

		// Editor Colors
		"editorBackground": "#000c18",
		"editorForeground": "#6688cc",
		"editorCursor": "#ddbb88",
		"editorWhitespaces": "#103050",
		"editorLineHighlight": "#082050",
		"editorSelection": "#770811",
		"editorIndentGuides": "#002952",
		"editorHoverBackground": "#000c38",
		"editorHoverBorder": "#004c18",
		"peekViewResultsBackground": "#060621",
		"peekViewResultsMatchForeground": "#7777cc",
		"peekViewResultsSelectionBackground": "#070866",
		"peekViewResultsSelectionForeground": "#7799ee",
		"peekViewEditorBackground": "#001F33",
		"peekViewTitleBackground": "#060621",
		"peekViewBorder": "#7777cc",
		"peekViewEditorMatchHighlight": "#ddbb5555",

		"editorLineNumbers": "#406385",
		"editorMarkerNavigationBackground": "#060621",
		"editorMarkerNavigationError": "#FF0000",
		"editorMarkerNavigationWarning": "#00FF00",

		"editorFindWidgetBackground": "#262641",

		// Workbench Colors
		"tabsContainerBackground": "#1c1c2a",
		"inactiveTabBackground": "#10192c",
		"tabBorder": "#2b2b4a",
		"editorGroupBorder": "#2b2b4a",
		"editorGroupBackground": "#1c1c2a",
		"editorDragAndDropBackground": "#25375daa",
		"activityBarDragAndDropBackground": "#25375daa",
		"editorSideBySideBorder": "#10192c",
		"panelTopBorder": "#2b2b4a",
		"statusBarBackground": "#10192c",
		"statusBarNoFolderBackground": "#10192c",
		"statusBarDebuggingBackground": "#10192c",
		"debugToolBarBackground": "#051336",
		"activityBarBackground": "#051336",
		"activityBadgeBackground": "#0063a5",
		"sideBarBackground": "#060621",
		"titleBarActiveBackground": "#10192c",
		"titleBarInactiveBackground": "#10192caa"
	}
```

**Comment by equinusocio** [comment]
And here
https://github.com/Microsoft/vscode/issues/3112#issuecomment-291063272

**Comment by kurtextrem** [comment]
Maybe this helps you: https://github.com/Tyriar/vscode-theme-generator it supports the bench themeing already.

**Comment by equinusocio** [comment]
Thank you, we are building our generator. Check https://github.com/equinusocio/vsc-material-theme/issues/13

**Comment by kurtextrem** [comment]
There are some settings for `settings.json` too: https://una.im/code-workspaces/

**Comment by kurtextrem** [comment]
May I ask what happened between 0.11 and 0.10? 0.10 had my Code completely themed, while 0.11 removed some tab styles, sidebar and so on.

**Comment by equinusocio** [comment]
0.0.10 was released for vscode 1.12 that isn't released yet, my fault. So 0.0.11 bring back old theme settings for vscode `< 1.12`. From vscode 1.12 the workbench theme setting are completely different so there is not backward compatibility. I will release the new version when vsc 1.12 come out.

**Comment by imarrsh** [comment]
Is there something in the works (or already existing) such as a contrast mode? Sort of like Sublime Text's Material Theme options (`"material_theme_contrast_mode": true`), but in VS Code where one could set `"workbench.colorTheme.contrastMode": true` so that the background color can be broken up a bit - perhaps a darker sidebar and such?

**Comment by equinusocio** [comment]
The contrast mode will be implemented after 1.0.0. If you want you can open a new related issue for tracking.
